target/ppc/translate/vmx-impl.c.inc

```diff
@@ -1284,6 +1284,39 @@ static bool trans_VINSERTD(DisasContext *ctx, arg_VX_uim4 *a)
     return true;
 }
 
+static bool do_vinsvx(DisasContext *ctx, arg_VX *a, int size, bool right,
+                      void (*gen_helper)(TCGv_ptr, TCGv_ptr, TCGv_i32))
+{
+    TCGv_i32 index;
+    TCGLabel *l0;
+
+    REQUIRE_INSNS_FLAGS2(ctx, ISA310);
+    REQUIRE_VSX(ctx);
+
+    index = tcg_temp_local_new_i32();
```

Comment:
Similar to what was done for vextdu*, can we get rid of the branch and check ``index`` inside the helper?